packages/components/src/buttons/src/Button.tsx

```diff
@@ -195,26 +110,6 @@ function Button(props: ButtonProps, ref: ForwardedRef<HTMLElement>) {
         return <EnsureTextWrapper>{prev}</EnsureTextWrapper>;
```

Comment:
FYI not important for this PR, but i don't like this being a componetn now, i prefered a function or manually do it

Reply:
Why is that? you mean keep it in utils but change to just a function?

Reply:
yeah